secrets.py

```diff
@@ -0,0 +1,26 @@
+"""
+Contains all the secrets for the website. Production uses a different file, that is not pushed to github.
+"""
+
+import json
+from os import path
+
+default_secrets = {
+    "SECRET_KEY": "j@7*rtssewjfhix2f^7&1iypigm=o4ju1qtdd!)ad$s1*hlkj2",
+    "DEBUG": True,
+    "ALLOWED_HOSTS": [],
+    "DATABASE_NAME": "cosmos_website_test",
+    "DATABASE_USER": "cosmos_website_tester",
+    "DATABASE_PASSWORD": "2020123",
+    "DATABASE_HOST": "localhost",
+    "DATABASE_PORT": "",
+}
+
+
+def get_secret(secret):
+    if path.exists("/etc/secrets.json"):
+        with open("/etc/secrets.json", "r") as f:
+            secrets = json.load(f)
+            return secrets[secret]
+    else:
+        return default_secrets[secret]
```

Comment:
Minor nitpick, is there a way to cache the file instead of opening the file every single time the function is called?

Reply:
This function is called when the server is starting but it will be called for every field. Then it would be better to cache it and delete the object from memory a minute after it was opened.